docs/about/products.md

```diff
@@ -0,0 +1,72 @@
+.NET Products
+=============
+
+.NET is a very flexible and naturally cross-platform [specification](https://github.com/dotnet/coreclr/blob/master/Documentation/project-docs/dotnet-standards.md) for building developer products. It is used for all of the most popular app categories: desktop, mobile, cloud, gaming and IoT. 
+
+Major commercial and open source .NET products are listed below, in order of introduction.
```

Comment:
I vote for the current "in order of introduction" as I think the number of people coming here who have never heard of .NET is likely to be small, while the number who have heard enough to think that ".NET = .NET Framework" is more.  Later when .NET Core is established we could revisit the ordering.